resources/assets/less/bem/wiki-main-page.less

```diff
@@ -38,4 +38,30 @@
 
     padding-top: 45px;
   }
+
+  &__heading {
+    color: @osu-colour-c1;
+    font-size: 20px;
+    font-weight: 300;
+    font-style: normal;
+
+    .wiki-main-page-panel & {
+      margin-bottom: 40px;
+    }
+
+    .wiki-main-page-panel:first-child & {
```

Comment:
should be a modifier that categorizes why it's `font-size` should be different instead of `:first-child`